jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/resources/org/eclipse/lsp4jakarta/jdt/core/messages/messages.properties

```diff
@@ -0,0 +1,16 @@
+# /*******************************************************************************
+# * Copyright (c) 2022 IBM Corporation and others.
+# *
+# * This program and the accompanying materials are made available under the
+# * terms of the Eclipse Public License v. 2.0 which is available at
+# * http://www.eclipse.org/legal/epl-2.0.
+# *
+# * SPDX-License-Identifier: EPL-2.0
+# *******************************************************************************/
+
+# NLS_MESSAGEFORMAT_VAR
+# NLS_ENCODING=UNICODE
+
+# Diagnostic messages
+AnnotationMustDefineAttribute = The annotation {0} must define the attribute ''{1}''.

```

Comment:
Is it possible to check the file in without the control-Ms? You may need to adjust your git configuration.

Reply:
Same issue in jakarta.ls.

Reply:
Not sure if we should do it using git configuration, as git could corrupt a lot of other files when git incorrectly "guesses" the line-endings or character is there but NOT as a line-ending.

Reply:
We have issue #321 to track this issue.